packages/core/src/components/graphs/line.ts

```diff
@@ -51,25 +51,27 @@ export class Line extends Component {
 				if (value === null || value === undefined) {
 					return false;
 				}
-
 				return true;
 			});
 
 		let data = [];
 		if (this.configs.stacked) {
-			const percentage = Object.keys(options.axes).some(axis => 
-				options.axes[axis].percentage	
-			)
+			const percentage = Object.keys(options.axes).some(axis =>
+				options.axes[axis].percentage
+			);
+			const { groupMapsTo } = options.data;
 			const stackedData = this.model.getStackedData({ percentage });
+			const domainIdentifier = this.services.cartesianScales.getDomainIdentifier();
+			const rangeIdentifier = this.services.cartesianScales.getRangeIdentifier();
 
 			data = stackedData.map((d) => ({
-				name: d[0].group,
+				name: d[0][groupMapsTo],
 				data: d.map((datum) => ({
-					date: datum.data.sharedStackKey,
-					group: datum.group,
-					value: datum[1],
+					[domainIdentifier]: datum.data.sharedStackKey,
+					group: datum[groupMapsTo],
```

Comment:
we don't need to do `groupMapsTo: datum[groupMapsTo]` here?

Reply:
👍  yeah youre right